tests/integration/runner.py

```diff
@@ -384,7 +384,7 @@ def _get_container_host_port(self, id_or_name, guest_port):
         )
         if not result[1]:
             raise RuntimeError("Cannot get container port: {}".format(result[0]))
-        return int(result[0].split(":")[1].rstrip())
+        return int(result[0].split(":")[-1].rstrip())
```

Comment:
Just curious to know, what changed in the sequence that now we pick the last?

Reply:
Not exactly sure what changed in the underlying platform where the tests are run, but I noticed the `docker port` output seems to have changed.
So this is a fix to address the failure [here](https://github.com/mendix/cf-mendix-buildpack/actions/runs/6119616281/job/16614934821).